### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by tondef001**
### Summary

Updated the template.  Ironically I didn't get to use my updated template for this PR.
The numbered steps are now numbered properly
The link is set to TFF instead of S002
### Checklist
- [x] Did you run `npm test`? NO
- [x] Did you update/add documentation for new methods or changed functionality? NO

Finally, is it ready to be reviewed and merged: :white_check_mark:
### How to Review
1. Look at the PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,15 +15,14 @@ Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red
 ### How to Review
 
 1. Put
-1. the steps
-1. to verify
-1. the PR's
-1. new functionality
-1. here
+2. the steps
```
Comment (mattmahn) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: No

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -15,15 +15,14 @@ Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red
...
+5. new functionality
+6. here
 
 ### Links
 
 Put links to Jira tickets, and anything else, here.
 
-- [S002-29](https://msoese.atlassian.net/browse/S002-29)
-
+- [TFF-29](https://msoese.atlassian.net/browse/TFF-29)
```
Comment (mattmahn) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: Yes

**Review by mattmahn — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (tondef001) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Why would you want the steps misnumbered in the template
Literally more work

Comment (mattmahn) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: its literally less work since it renumbers it when rendered

**Pull request merged**